src/ui/Video/Video.stories.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Container } from 'semantic-ui-react';
+import NextCloudEmbed from '@eeacms/volto-nextcloud-video-block/NextCloud/players/NextCloudEmbed';
+
+export default {
+  title: 'Components/Video',
+  component: NextCloudEmbed,
+  argTypes: {},
+};
+
+const Template = (args) => (
+  <Container>
+    <div className="block video align center">
+      <div className="video-inner">
+        <NextCloudEmbed {...args} />
+      </div>
+    </div>
+  </Container>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  data: {
+    url: 'https://cmshare.eea.europa.eu/s/Xw7kH2TJGCibHJf/download',
+    autoPlay: false,
+    loop: false,
```

Comment:
@avoinea I have them loop & autoplay here inside a data object, as I saw it here:

https://github.com/eea/volto-nextcloud-video-block/blob/eeb32ef3ae94325b94cf68cdc73e1e99671016f4/src/NextCloud/players/NextCloudEmbed.jsx#L27

let me check the hard dependancy and if I can move them outside the data object